guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -34,3 +34,9 @@ ifdef::satellite,orcharhino[]
 This migration takes time, so a pre-migration runs automatically after the upgrade to {ProjectVersion} to reduce future upgrade downtime.
 While the pre-migration is running, {ProjectServer} is fully functional but uses more hardware resources.
 endif::[]
+ifdef::satellite[]
+ifeval::["{mode}" == "connected"]
+. If you use Red[nbsp]Hat Insights, re-register the {ProjectServer} to Insights.
```

Comment:
```suggestion
. If you use Red{nbsp}Hat Insights, re-register your {ProjectServer} to Insights.
```

blocking: Please use curly brackets for attributes such as "nbsp".

non-blocking: Please use "your Foreman Server" in favor of "the Foreman Server". Applies also to line 16 and 18 in the other file.